pulseguardian/config.py

```diff
@@ -12,11 +12,11 @@
 
 flask_debug_mode = bool(int(os.getenv('FLASK_DEBUG_MODE', 1)))
 
-# Persona
-persona_verifier = os.getenv('PERSONA_VERIFIER',
-                             'https://verifier.login.persona.org/verify')
-persona_audience = os.getenv('PERSONA_AUDIENCE',
-                             'https://{0}:{1}'.format(flask_host, flask_port))
+# Auth0
+auth0_client_secret = os.getenv('AUTH0_CLIENT_SECRET', None)
+auth0_client_id = os.getenv('AUTH0_CLIENT_ID', 'wLkelObcNuywt1X1uxGNeFvdKsLY0dSb')
```

Comment:
Yeah, without the secret, this is probably safe and all, but maybe I shouldn't include it as a default value.  The user will have to go to Heroku to get the env var value for the secret to use auth locally anyway.  So probably better to leave it off.